apps/quick-dapp/src/actions/index.ts

```diff
@@ -192,7 +193,9 @@ export const deploy = async (payload: any, callback: any) => {
     files[`dir/${path}`] = resp.data;
   }
 
-  files['dir/assets/logo.png'] = logo
+  if (logo) {
```

Comment:
there is no default case anymore?

Reply:
That's it. No more default logo. @yann300